blog/admin.py

```diff
@@ -0,0 +1 @@
+"""This module registers models with the Django admin."""
```

Comment:
The module doesn't *DO* anything. You do, the code does. If it's not an entry-point script, that is.
Modules aren't functions. So they aren't actions by themselves. They are objects/things. Describe what they are, not what they do.

Reply:
Edited. Thank you.

Reply:
Marking as resolved.